### ISSUE TITLE
Support persistent tor profile

### ISSUE THREAD

**Issue by darkdh** [brave/brave-browser#1115]
With this feature, user can generate many Tor profiles for different purposes in Profile Manager. 
And also remove any of tor profiles so any data(history, cookies, bookmark...etc) within the profile will be wiped out.

This can supersede current Tor window feature because each Tor profile has its own incognito window.

**Comment by srirambv** [comment]
Extensions enabled to work in Incognito mode should work on Tor profiles as well

**Comment by petemill** [comment]
As worded, I believe this still won’t support using bookmarks (or extensions) from the main profile. It would have to be a new profile, somehow synced or backed up differently.

**Comment by darkdh** [comment]
@petemill yes, completely isolated profile. If we want to inherit something from certain profile, that would be session window scope. @bridiver was working on that.  I agree that setting up everything for each new profile is time consuming but tor profile shouldn't inherit anything from non tor profiles, we can let users to choose what tor profile to inherit from when generating new tor profile or they want a brand new one.

**Comment by jonathansampson** [comment]
@darkdh  Would this feature allow the user to Sync bookmarks (and other data, eventually) between a standard profile, and a Tor profile?

**Comment by darkdh** [comment]
like https://github.com/brave/brave-browser/issues/1115#issuecomment-422874906 said. It will inherit data from the profile which opened the tor window. Just like one time session windows but for tor

**Comment by tomlowenthal** [comment]
Just to be clear, this isn't the mainstream roadmapped feature for migrating Tor windows from Guest to regular Incognito sessions. This is an additional, separate feature request which is not currently planned for implementation.

**Comment by bsclifton** [comment]
This issue hasn't had an update in a while - it's met the criteria for being stale. The issue is going to be closed for now. If this is still a problem, please leave a comment or re-open the issue. Thanks!
